config/longrun_configs/longrun_amip_dyamond.yml

```diff
@@ -1,12 +0,0 @@
-aerosol_radiation: true
-atmos_config_file: "config/longrun_configs/longrun_aquaplanet_dyamond.yml"
```

Comment:
Can we switch all `AMIP` configs to use one of either   
```
amip_target_diagedmf.yml
amip_target_edonly.yml
``` 
Tracking `toml` files for parameters is then simpler.